src/components/CommentsCard.jsx

```diff
@@ -6,22 +6,36 @@ import Comment from "./Comment";
 import CommentForm from "./CommentForm";
 
 const CommentsCard = () => {
+  const loggedInUser = "grumpy19";
   const { article_id } = useParams();
   const [comments, setComments] = useState([]);
+
   const addComment = (text) => {
     axios.post(
       `https://nc-news-czlb.onrender.com/api/articles/${article_id}/comments`,
-      { username: "grumpy19", body: text }
+      { username: loggedInUser, body: text }
     );
   };
+
+  const deleteComment = (comment.comment_id) => {
```

Comment:
it's incorrect syntax to try pass a property into this deleteComment function - we'd want this to expect the commentId directly at this point instead, (you can pass comment.comment_id when it's invoked, but not when you're first declaring the function)